### PR TITLE
Fix bug 1486516 - Continuous loading of entities. 

### PR DESCRIPTION
**Pull request by adngdb**
This makes sure that the screen always takes 100% of the screen, and that scrolling is handled in panels, notably in the EntitiesList component. It also adds an infinite scroller so that next entities are loaded when a user goes near the bottom of the list (by default, within 250px from the bottom).

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #app {
     display: flex;
+    height: 100%;
 }
```
Comment (mathjazz) [inline frontend/src/App.css]: Stupid question: can we merge `#root` and `#app` into a single element?

File: frontend/src/modules/entitieslist/actions.js [hunk 2]
```diff
@@ -84,13 +92,25 @@ export function get(
 
         const response = await fetch(url, requestParams);
         const content = await response.json();
-        dispatch(receive(content.entities));
+        dispatch(receive(content.entities, content.has_next));
```
Comment (mathjazz) [inline frontend/src/modules/entitieslist/actions.js]: It's a bit confusing to have `has_next` and `hasMore` - should we unify that?

Comment (adngdb) [reply frontend/src/modules/entitieslist/actions.js]: `<InfiniteScroll>` accepts that argument as `hasMore`, which is why I called in that in my code, and the API exposes `has_next`. I don't want to mess with the API as part of Translate.Next (not until we're done with rewriting everything), so we're bound to having this difference. The question is, which term do we think is better, and thus want to use in our code? I don't really have an opinion there, if you have one, please let me know.

**Comment by adngdb** [comment]
@mathjazz I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1489054 to add loaders. I want to keep PRs small, and this one is already too big in my opinion.

**Pull request merged**